feature_engine/feature_selection.py

```diff
@@ -0,0 +1,85 @@
+
+import warnings
+
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.utils.validation import check_is_fitted
+from feature_engine.dataframe_checks import _is_dataframe, _check_input_matches_training_df
+from feature_engine.variable_manipulation import _define_variables
+
+
+class DropFeatures(BaseEstimator, TransformerMixin):
+    """
+    The FeatureEliminator() drops the list of variable(s) as provided by the user
+    from the dataframe and returns the subset of original dataframe with remaining
+    variables.
```

Comment:
DropFeatures() drops the list of variable(s) indicated by the user
    from the original dataframe and returns the remaining variables.

Could we replace doscstrings by the above?